net/znc/Makefile

```diff
@@ -31,6 +31,7 @@ define Package/znc/default
   CATEGORY:=Network
   TITLE:=ZNC
   URL:=http://en.znc.in/
+  USERID:=znc=65532:znc=65532
```

Comment:
Just in case it may help, there is no need to determine the id at packaging time.  It can be simply `USERID=znc:znc` and opkg will decide a free uid/gid pair at `postinst` stage.